server/src/routes/api/v1/parkReviewsRouter.js

```diff
@@ -34,4 +34,31 @@ parkReviewsRouter.post("/", async (req, res) => {
   }
 });
 
+parkReviewsRouter.delete("/:reviewId", async (req, res) => {
+  try {
+    const reviewId = req.params.reviewId;
+    await Review.query().deleteById(reviewId);
+    return res.status(204).json({});
+  } catch (error) {
+    return res.status(500).json({ errors: error });
+  }
+});
+
+parkReviewsRouter.patch("/:reviewId", async (req, res) => {
+  try {
+    const parkId = req.params.parkId;
+    const reviewId = req.params.reviewId;
+    const comments = req.body.comments;
```

Comment:
Just want to mention here that the column `comments` on the Review table is a bit confusing since the name is plural. We would think that a Review contains a rating and _a_ comment (description, body, text, etc). This doesn't need to be updated, just something to keep in mind!